hal/src/main/native/shared/cpp/DriverStation.cpp

```diff
@@ -5,6 +5,8 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
+#include "HAL/DriverStation.h"
```

Comment:
Why the reorder here. If for an include issue, we should fix that by making the bad header fully self contained.

Reply:
I am trying to be rigorous in running wpiformat; that was a change it prompted.

Reply:
Ah. I didn't know if it was a build required change or a wpiformat change. If it was a wpiformat change, carry on.